static/js/components/ErrorMessage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import LoginButton from '../containers/LoginButton';
+import { Alert } from 'react-bootstrap';
```

Comment:
It's generally better to import from the specific file so we don't necessarily include all of `react-bootstrap` for a single component:

    import Alert from 'react-bootstrap/lib/Alert';